.github/workflows/ci.yaml

```diff
@@ -284,7 +285,8 @@ jobs:
         run: |
           mkdir -p cache
           echo "Hello World" > cache/hello.txt
-          echo "::set-output name=success::true"
+          # shellcheck disable=SC2086
+          echo "success=true" >> $GITHUB_OUTPUT
```

Comment:
You can quote the "$GITHUB_OUTPUT" in lieu of disabling shellcheck for the quoting.

Reply:
Good catch, I did that pretty much everywhere else. This due to my early efforts at this refactoring. In any case, I have cleaned this up as you suggest.